### DISCUSSION TITLE
Proof of Contribution in social thesis

### DISCUSSION THREAD

**Discussion by 0xsingular** [teaxyz/white-paper#110] [Ideas]
As far as we know as a simple form the tea protocol will focus on (financial) support for developers working/contributing to open source projects.
Are there plans for the future or the idea of adding another side to the tea protocol that will allow projects to receive social, community support and community involvement in building and supporting projects?

This way developers who come up with an original concept that is needed by the web3 ecosystem in general will have full support for the realisation of this project. The social part should not include TEA rewards, already each project in an open form can set the benefits for social contributions. It would probably be achievable through a combination of Proof of Contribution consensus and [syndicate’s programmable smart wallets](https://docs.syndicate.io/features/programmable-wallets) + [modular smart contracts](https://docs.syndicate.io/features/smart-contracts). In theory such involvement would build a stronger community around the projects, and developers would be more excited to experiment with new ideas knowing that they would have a protocol (tea) through which they could connect with people who could provide the full support needed to realize these ideas.

**Comment by thomas-borrel** [comment]
Hi.
One of the goals of the protocol is to allow each OSS project to become its own DAO. This will first materialize with each project being associated with a treasury (that receives rewards and donations). Signing contributors can decide to whom and under what conditions assets are being transfered. Projects will also be able to establish their own grants program. 

We believe that empowering each project to become a DAO is important and could help foster the behavior and create the opportunities you described. That said, we believe the protocol must focus on providing tools and let each project decide how they want to be governed or how they want to engage with their own community. 

Curious to hear your thoughts.

**Reply by 0xsingular** [reply]
The idea of being able to become a DAO for each OSS project is even better, it gives more flexibility to each project in achieving technological and growth milestones.
Thank you for your time!
I'll be here because I sure love what tea.xyz is and what it's going to be.

**Reply by thomas-borrel** [reply]
Awesome! Glad you like it.
Keep the feedback coming. It's important to hear what you think could be done better.
